flow/envs/base_env.py

```diff
@@ -323,12 +323,6 @@ def setup_initial_state(self):
             value = sparse state information (only what is needed to add a
             vehicle in a sumo network with traci)
         """
-        # check to make sure all vehicles have been spawned
-        num_spawned_veh = self.traci_connection.simulation.getDepartedNumber()
```

Comment:
why is this a bad check to have in place?

Reply:
Don’t worry, it was just shifted down. The check is still there.